### PR TITLE
Pull request for adding person, student and teacher classes.

### PR DESCRIPTION
**Pull request by TechnoAndy**
In this project, I implemented the following:

 - Created classes for person, student and teacher.
 - Created relevant constructors 
 - Each class is in a separate file

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative('./person')
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```
Comment (julie-ify) [inline student.rb]: - [ ] Kindly note that the `parent_permission` argument uses a different notation (:) so its assignment would be different. Kindly update your super parameters to this.

  ```
  super(age, name, parent_permission: parent_permission)
  ```

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require_relative('./person')
+
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```
Comment (julie-ify) [inline teacher.rb]: - [ ] Kindly note that the `parent_permission` argument uses a different notation (:) so its assignment would be different. Kindly update your super parameters to this.

  ```
  super(age, name, parent_permission: parent_permission)
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @TechnoAndy,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (TechnoAndy) [reply student.rb]: Thank you, I will do so

File: teacher.rb (hunk 2 above)
Comment (TechnoAndy) [reply teacher.rb]: Thank you, I will do so

**Review by mckent05 — APPROVED** [review]
Hi @TechnoAndy, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**